src/index.ts

```diff
@@ -10,6 +10,7 @@ if (!basicTool.getGlobal("Zotero")[config.addonInstance]) {
 	defineGlobal("console");
 	defineGlobal("performance");
 	defineGlobal("ZoteroPane");
+	defineGlobal("IntersectionObserver");
```

Comment:
Is this needed for the row line height resizing to work? Maybe could you add a comment about it here?